src/backend/executor/CallInstr.js

```diff
@@ -0,0 +1,30 @@
+define(["./FunctionCall"], function(functionCall) {
+
+    var callFunction = functionCall.callFunction;
+
+    function CallInstr(std) {
+        this.std = !!std;
```

Comment:
Why convert that to boolean? It could be funny if you pass wrong value and it gets quietly passed as true.